pymc3/examples/rugby_analytics.py

```diff
@@ -0,0 +1,132 @@
+
+# coding: utf-8
+
+# # A Hierarchical model for Rugby prediction
+
+# # Based on http://danielweitzenfeld.github.io/passtheroc/blog/2014/10/28/bayes-premier-league/
+
+
+import numpy as np
+import pandas as pd
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+import pymc3 as pm3, theano.tensor as tt
```

Comment:
I wonder if we even need the .py file, it's bound to just get out-of-sync and we'll have to update in two places if API changes.